### PR TITLE
Add df -> arrow table benchmark and chicago traffic dataset

### PR DESCRIPTION
**Pull request by jonkeane**

### CODE REVIEW

File: R/bm-R-to-arrow.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#' Benchmark file writing
+#'
+#' @section Parameters:
+#' * `source` A known-file id, or a CSV(?) file path to read in
+#' * `format` One of `c("parquet", "feather", "fst")`
+#' * `compression` One of `c("uncompressed", "snappy", "zstd")`
+#' * `input` One of `c("arrow_table", "data_frame")`
+#'
+#' @export
+df_to_table <- Benchmark("df_to_table",
```
Comment (nealrichardson) [inline R/bm-R-to-arrow.R:10]: We should name the file `bm-df-to-table.R` to match the Benchmark name

File: R/bm-R-to-arrow.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  setup = function(source = names(known_sources), ...) {
```
Comment (nealrichardson) [inline R/bm-R-to-arrow.R:11]: remove ... now?

File: R/bm-R-to-arrow.R [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#' Benchmark file writing
+#'
+#' @section Parameters:
+#' * `source` A known-file id, or a CSV(?) file path to read in
+#' * `format` One of `c("parquet", "feather", "fst")`
```
Comment (nealrichardson) [inline R/bm-R-to-arrow.R:5]: These params don't seem to match, nor does the docstring title

**Review by nealrichardson — CHANGES_REQUESTED** [review]

File: R/bm-R-to-arrow.R (hunk 2 above)
Comment (jonkeane) [reply R/bm-R-to-arrow.R:11]: Yes, thanks for catching that. I had removed all of them on the _other_ branch but not here

**Comment by jonkeane** [comment]
This should be ready to go, any objections to merging?

**Pull request merged**